src/constants/server/props.ts

```diff
@@ -0,0 +1,8 @@
+import { PAGE_PATH } from 'constants/common';
+
+export const REDIRECT_LOGIN_PAGE_PROPS = {
+  redirect: {
+    destination: PAGE_PATH.account.login,
+    permanent: false,
+  },
+};
```

Comment:
p3;
혹시 요런 방식 어떻게 생각하세요?
하나의 객체로 연관된 상수를 관리하면 좋을 것 같아서요!

```suggestion
export const SERVER_SIDE_PROPS = {
  REDIRECT_LOGIN: {
    redirect: {
      destination: PAGE_PATH.account.login,
      permanent: false,
    }
  },
};
```

Reply:
너무 좋습니다!! 추가로 props가 생기더라도 어떤 props를 사용할 지 찾기가 더 쉬울 것 같네요 
좋은 의견 감사합니다 [2d7d115](https://github.com/a-daily-diary/ADD.FE/pull/246/commits/2d7d115e6b18f611d9d6972d1277061cf3308c92)